glossary.rst

```diff
@@ -0,0 +1,46 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This page contains definitions for Flux-specific terms and concepts mentioned
+throughout the various guides and docs on our sites.
+
+------------------------------------------------------------------------------
+
+**Flux Broker**
+  A distributed message broker daemon that provides communications services
+  within a Flux instance.
+
+**Flux Instance**
+  A group of flux-broker processes which are launched via any parallel launch
+  utility that supports PMI (Process Manager Interface).
+
+**Hostlist**
```

Comment:
Can we show an example here? I would say it's a list of hosts with patterns that are known how to be expanded (and show an example).

Reply:
Sure! That sounds like a good suggestion to me. Are you thinking of an example at the end of the definition like the following? "Example: `my_host[1-10]`"

Reply:
Yes exactly! Say something like `my_host[1-10]` expands to `my_host1` `my_host2` ... `my_host10`

Reply:
One or more of the [test vectors](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_29.html#test-vectors) from the RFC might work?